language/functional_tests/tests/testsuite/block/expired_transaction.mvir

```diff
@@ -0,0 +1,62 @@
+//! account: vivian, 1000000, 0, validator
+
+//! new-transaction
+import 0x0.LibraSystem;
+
+main() {
+    // TODO: this will stop working once we make the Block prologue private. We should delete this
+    // part (which will be done by the executor) and leave the second assert
+    LibraSystem.block_prologue(100000000, h"aa", h"bb", {{vivian}});
+    return;
+}
+
+//! new-transaction
+//! expiration-time: 100
+import 0x0.LibraSystem;
+import 0x0.LibraTimestamp;
+
+main() {
+    return;
+}
+// check: TRANSACTION_EXPIRED
+
+//! new-transaction
+//! expiration-time: 700
+import 0x0.LibraSystem;
+import 0x0.LibraTimestamp;
+
+main() {
+    return;
+}
+// check: TRANSACTION_EXPIRED
+
+//! new-transaction
+// 100 + 600 < 701, should be accepted.
```

Comment:
I don't understand this computation--don't we need `expiration_time` * 1000 > 100000000 (the value passed to `block_prologue` above?)